examples/cardano-explorer-api-migration/genesis/getGenesisAddressInfo.graphql

```diff
@@ -0,0 +1,21 @@
+# TODO: this doesn't really map to the REST response type,
```

Comment:
I'm not really sure how useful this query is, since it's really very different from the REST output.